### PR TITLE
Change existing DTO in Create Handlers to  Create DTO without id. Issue 1343

### PR DESCRIPTION
**Pull request by ValDekh**
## Summary of change

Change existing DTO in Create Handlers to  Create DTO without id

### CODE REVIEW

File: Streetcode/Streetcode.WebApi/Extensions/SeedingLocalExtension.cs [hunk 1]
```diff
@@ -35,8 +35,8 @@ public static async Task SeedDataAsync(this WebApplication app)
                 string blobPath = app.Configuration.GetValue<string>("Blob:BlobStorePath");
                 var repo = new RepositoryWrapper(dbContext);
                 var blobService = new BlobService(blobOptions, repo);
-                string initialDataImagePath = "../Streetcode.DAL/InitialData/images.json";
-                string initialDataAudioPath = "../Streetcode.DAL/InitialData/audios.json";
+                string initialDataImagePath = "../../../TestData/InitialData/images.json";
+                string initialDataAudioPath = "../../../TestData/InitialData/audios.json";
```
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Extensions/SeedingLocalExtension.cs:39]: Maybe create some sort of variable for path? Just a suggestion, maybe it is better this way

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: Add new line at the end of file

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public CreateTermHandlerTests()
 			_mockMapper = new();
 			_mockRepository = new();
             _mockLogger = new Mock<ILoggerService>();
-            _mockLocalizerCannotConvertNull = new Mock<IStringLocalizer<CannotConvertNullSharedResource>>();
+			_mockLocalizerCannotConvertNull = new Mock<IStringLocalizer<CannotConvertNullSharedResource>>();
```
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs:30]: Why the long space?)

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs [hunk 3]
```diff
@@ -80,8 +80,9 @@ public async Task ShouldThrowException_WhenTryToAddNull(int returnNumber)
 		}
 
 		private static Term GetTerm() => new();
-		private static TermDTO GetTermDTO() => new();
-		private static Term? GetNotExistingTerm() => null;
-		private static TermDTO? GetNotExistingTermDTO() => null;
+		private static TermCreateDTO GetTermCreateDTO() => new();
+        private static TermDTO GetTermDTO() => new();
+        private static Term? GetNotExistingTerm() => null;
```
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs:85]: Formating error?

**Review by Lazy-Lenny — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.BLL/DTO/Media/Images/ImageDTOCreateEntities.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Streetcode.BLL.DTO.Media.Images
+{
+    public class ImageDTOCreateEntities
```
[the remark this answers is not in the document]
Comment (ValDekh) [reply Streetcode/Streetcode.BLL/DTO/Media/Images/ImageDTOCreateEntities.cs:3]: Fix

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Fact/Create/CreateFactCommand.cs
[the remark this answers is not in the document]
Comment (ValDekh) [reply Streetcode/Streetcode.BLL/MediatR/Streetcode/Fact/Create/CreateFactCommand.cs:1]: Fix

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs
Comment (ValDekh) [reply Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: Need some clarification

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs (hunk 2 above)
Comment (ValDekh) [reply Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs:30]: image
Everything correct

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs
Comment (Lazy-Lenny) [reply Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: If we look at the raw representation of files, for example:
image
This is good!
Buuuut:
image
This is not.
Also github tells us when we forgot new line at the end of file. It's marked as red sign at the very end of such file
Like this:
image
Additional sources:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

File: Streetcode/Streetcode.BLL/Mapping/Streetcode/TextContent/TermProfile.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.BLL/Mapping/Streetcode/TextContent/TermProfile.cs:1]: Add new line at the end of file

File: Streetcode/Streetcode.BLL/Mapping/Streetcode/TextContent/TextProfile.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.BLL/Mapping/Streetcode/TextContent/TextProfile.cs:1]: Add new line at the end of file

File: Streetcode/Streetcode.BLL/MediatR/Streetcode/Fact/Create/CreateFactCommand.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.BLL/MediatR/Streetcode/Fact/Create/CreateFactCommand.cs:1]: Add new line at the end of file

File: Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs
Comment (Lazy-Lenny) [inline Streetcode/Streetcode.WebApi/Extensions/ServiceCollectionExtensions.cs:1]: Add new line at the end of file

**Review by Lazy-Lenny — APPROVED** [review]

**Review by Lazy-Lenny — APPROVED** [review]

**Pull request closed without merging**